### DISCUSSION TITLE
How can I hide/disable the "on this page" on mobile devices?

### DISCUSSION THREAD

**Discussion by Ali-Hussein-dev** [fuma-nama/fumadocs#933] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/756a8542-b521-4d64-89fb-f54b323d4ccc)

the tableOfContent is disabled though, 

```tsx
 <DocsPage
      tableOfContent={{
        enabled: false,
      }}
      full={true}
    >
     // ...
    </DocsPage>
```

**Comment by Gitstar-OC** [comment]
Is it still showing on mobile ?

**Reply by Gitstar-OC** [reply]
Means after disabling the toc on pc.

**Reply by Ali-Hussein-dev** [reply]
yes

**Reply by fuma-nama** [reply, marked as answer]
It's the `tableOfContentPopover` option, See https://fumadocs.vercel.app/docs/ui/blocks/page#popover-mode

**Reply by GermanJablo** [reply]
How can I disable "on this page" on desktop?

I tried the `header` property of `tableOfContent` and `tableOfContentPopover`, but nothing changed. I don't have a problem on mobile because I don't see it.

A property would be ideal, but I don't see an ID for CSS either.

**Reply by fuma-nama** [reply]
`tableOfContent.enabled`

**Reply by GermanJablo** [reply]
But that disables the entire ToC, right? I just want to remove the "on this page" part. Similar to what Clerk does. I think it's cleaner.

<img width="297" height="550" alt="image" src="https://github.com/user-attachments/assets/f1b7e6e9-d98c-4dd5-8f41-45683019b7ec" />

**Reply by fuma-nama** [reply]
You can replace the labels with translations API, but in your case hiding it with CSS would also be enough

**Reply by GermanJablo** [reply]
There really wasn't a way to hide it with Translations API. The best thing I could do was change it to an empty string but the icon and space remained visible.

Also, it wasn’t easy to hide it with CSS because it didn’t have any specific selector. 

I've [created a PR](https://github.com/fuma-nama/fumadocs/pull/2472) that adds a `showTitle` option and also adds an id to the dom element.

**Comment by Gitstar-OC** [comment]
![Screenshot 2024-10-06 134737](https://github.com/user-attachments/assets/9094f919-ba3f-4d84-b04b-789a96b3e05b)

You can hide it using css for the id of the div, it will be hidden. 
```css
#nd-tocnav {
visibility: hidden 
} 
```

There is no problem hiding it, it will also be passed down as metadata for better seo.

**Reply by Ali-Hussein-dev** [reply]
this is a hack, but it should be configurable

**Reply by Gitstar-OC** [reply]
It is configurable, I gave you incase it was urgent or causing any ui problem or glitch. It solves your hiding problem :)

**Reply by fuma-nama** [reply]
Yes there's an option to disable it from React, but yes, you can customise them with CSS selectors (that's one reason why we added the `id` attribute, it's mainly for custom styling)
